setup.py

```diff
@@ -41,19 +41,19 @@ def get_description():
 
 
 REQUIREMENTS = [
-    'spyder>=4.1.0,<5.0.0',
+    'spyder>=5.0.4',
     'tornado',
-    'terminado>=0.9.1',
+    'terminado>=0.10.0',
```

Comment:
Does `terminado` pull `pywinpty`? Or why do we not depend on it directly?

Reply:
Yes, terminado pulls pywinpty